packages/quill-next-vue/src/components/quill-editor.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+	<div ref="quill-container"></div>
+</template>
+
+<script setup lang="ts">
```

Comment:
I find out that you use a `\t` as an indent. For consistency, I'd like to propose we use two spaces for indentation, which matches Quill's existing style. To align with our current codebase will keep things clean. Could you please update the formatting on these files?